Documentation/ExtensionArchitecture/Extbase/Index.rst

```diff
@@ -13,21 +13,21 @@ Extbase is an extension framework to create TYPO3 frontend plugins and TYPO3
 backend modules. Extbase can be used to develop extensions but it does not
 have to be used.
 
-Extbase is included in the TYPO3 Core as system extension :php:`extbase`.
-
-**Please note:** Extbase relies on :ref:`frontend TypoScript <t3tsref:start>`
-being present; otherwise the configuration is not applied. This is usually no
-problem - Extbase plugins are typically either included as
-:ref:`USER content object <t3tsref:cobj-user>` (its content is cached and
-returned together with other content elements in fully-cached page context), or
-the Extbase plugin is registered as USER_INT. In this case, the
-TYPO3 Core takes care of calculating TypoScript before the plugin is
-rendered, while other USER content objects are fetched from page cache. This in
-mind you should not use Extbase in another context, like in
-:ref:`middlewares <request-handling-middlewares-extbase>`.
-
-There are also tutorials in the :ref:`Extension Development -
-Tutorials <extension-tutorials>` section.
+Extbase is a framework for developing TYPO3 extensions, providing a structured
```

Comment:
I would avoid a "block of text" by  using paragraphs and maybe even intermediate headlines

Reply:
Done. I have added "start" to all header to reduce concurrency with upcoming header in introduction and other pages